### DISCUSSION TITLE
Unable to listen database changes in supabase

### DISCUSSION THREAD

**Discussion by mustafakendiguzel** [supabase/supabase#28698] [Questions]
I can't listen to database changes in supabase, its also giving me error to when try to create channel 

<img width="308" alt="Ekran Resmi 2024-08-16 16 36 01" src="https://github.com/user-attachments/assets/a8f64f33-d955-498f-9f6b-1fea302ceea5">
<img width="1009" alt="Ekran Resmi 2024-08-16 16 38 11" src="https://github.com/user-attachments/assets/18e632cb-4647-4fe8-950b-ba77910af63a">
<img width="1009" alt="Ekran Resmi 2024-08-16 16 38 33" src="https://github.com/user-attachments/assets/76bf7795-c7f9-4a34-99ee-34582cfe17e7">

**Comment by GaryAustin1** [comment]
If that is a Supabase hosted instance you should contact support.
